pkg/kubelet/rkt/rkt.go

```diff
@@ -79,7 +79,7 @@ const (
 	k8sRktCreationTimeAnno           = "rkt.kubernetes.io/created"
 	k8sRktContainerHashAnno          = "rkt.kubernetes.io/containerhash"
 	k8sRktRestartCountAnno           = "rkt.kubernetes.io/restartcount"
-	k8sRktTerminationMessagePathAnno = "rkt.kubernetes.io/terminationMessagePath"
+	k8sRktTerminationMessagePathAnno = "rkt.kubernetes.io/termination-message-path"
```

Comment:
(expanding..) You're right. We should try to be consistent when we can. Modifying the keys later is not fun since maintaining backward compatibility is a pain.